rgw/v2/tests/s3_swift/test_swift_basic_ops.py

```diff
@@ -144,16 +146,57 @@ def test_exec(config, ssh_con):
     log.info(type(ceph_conf))
     rgw_service = RGWService()
     # preparing data
-    tenants_user_info = []
     user_name = names.get_first_name() + random.choice(string.ascii_letters)
-    tenant = "tenant"
-    tenant_user_info = umgmt.create_tenant_user(
-        tenant_name=tenant, user_id=user_name, displayname=user_name
-    )
-    tenants_user_info.append(tenant_user_info)
-    user_info = umgmt.create_subuser(tenant_name=tenant, user_id=user_name)
-    auth = Auth(user_info, ssh_con, config.ssl)
-    rgw = auth.do_auth()
+    if config.user_type == "non-tenanted":
+        users_info = []
```

Comment:
Lets add a method to create nontenanted user in reusable.py,
since we are using same workflow twice to create user and new_user